### DISCUSSION TITLE
用户关注取关的消息重复发送

### DISCUSSION THREAD

**Discussion by rsaLive** [w7corp/easywechat#2204] [Q&A]
$app->server->push(function($message){{
     //接受重复的用户关注取关的消息
})

**Comment by overtrue** [comment]
重复发送是微信的服务器行为哈：

> 微信服务器在五秒内收不到响应会断掉连接，并且重新发起请求，总共重试三次。
https://developers.weixin.qq.com/doc/offiaccount/Message_Management/Receiving_event_pushes.html
